### PR TITLE
Update w.glossary.md

### PR DESCRIPTION
**Pull request by Babazyga**
Added a list of terms related to the "Workflows"

### CODE REVIEW

**Comment by Babazyga** [comment]
@billy-the-fish can you please check it

File: docs/w.glossary.md [hunk 1]
```diff
@@ -21,4 +21,12 @@ Learn the significance of terminology commonly used in the documentation.
 #### End-user
  - An end-user of partner's application connected to Monite API. In the majority of cases this resembles an entity's employee.
  - The following terms might be used as synonyms and help your understanding: entity-user. 
- 
\ No newline at end of file
+#### Approval policy
+  - A single set of Rules and Conditions that was dedicated for user-defined purpose.
```
Comment (billy-the-fish) [inline docs/w.glossary.md:25]: A single set of [rules](#rule) and [conditions](#condition) dedicated for a user-defined purpose.  
The following terms are synonyms: [workflow](#<link>), [approval flow](#<link>), [spending policy](#<link>).

File: docs/w.glossary.md [hunk 2]
```diff
@@ -21,4 +21,12 @@ Learn the significance of terminology commonly used in the documentation.
...
+  - The following terms might be used as synonyms and help your understanding: Workflow / Approval flow / Spending policy
+#### Condition
+  - A setting that determines the situation for which an Approval policy should be applied.
```
Comment (billy-the-fish) [inline docs/w.glossary.md:28]: A setting that determines the situation in which an [approval policy](@approval-policy) is applied.

File: docs/w.glossary.md [hunk 3]
```diff
@@ -21,4 +21,12 @@ Learn the significance of terminology commonly used in the documentation.
...
+#### Rule
+  - A setting that defines what users should approve a submitted payable document in a set range of amounts.
```
Comment (billy-the-fish) [inline docs/w.glossary.md:30]: ... defines the users who have the rights to approve a submitted payable that contains figures in a specific range.

File: docs/w.glossary.md [hunk 4]
```diff
@@ -21,4 +21,12 @@ Learn the significance of terminology commonly used in the documentation.
...
+#### Approval permissions
+  - An authority of user in the Approval policy.
```
Comment (billy-the-fish) [inline docs/w.glossary.md:32]: I don't understand what you mean by authority.

in the [approval policy](#approval-policy)

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Nearly there, just a couple of updates. 

Global rule. In a glossary an mention of another term in the glossary must be a link to that term.

File: docs/w.glossary.md (hunk 1 above)
Comment (Babazyga) [reply docs/w.glossary.md:25]: Sorry, but how can I add a link on it?

File: docs/w.glossary.md (hunk 4 above)
Comment (Babazyga) [reply docs/w.glossary.md:32]: Yep, here I tried to say that "approval permissions" is a kind of "role". 

- For example, in Monite you are an admin
- But in "approval permissions" you are a "team owner"
- And if someone created a rule related to your team that goes like "If an amount more than 500 USD then it should be approved by team owner" then you should review all of the $500+ bills

Does it make sense?

File: docs/w.glossary.md (hunk 3 above)
Comment (Babazyga) [reply docs/w.glossary.md:30]: Thanks!

**Comment by Babazyga** [comment]
@billy-the-fish 
Iain, please check the changes then you'll have time

File: docs/w.glossary.md [hunk 5]
```diff
@@ -21,4 +21,12 @@ Learn the significance of terminology commonly used in the documentation.
 #### End-user
  - An end-user of partner's application connected to Monite API. In the majority of cases this resembles an entity's employee.
  - The following terms might be used as synonyms and help your understanding: entity-user. 
- 
\ No newline at end of file
+#### Approval policy
+  - A single set of [Rules](@rule) and [Conditions](@condition) that was dedicated for user-defined purpose.
```
Comment (billy-the-fish) [inline docs/w.glossary.md:25]: Links in markdowns are denoted with # not @.
Please, always use the present tense

 A single set of [Rules](#rule) and [Conditions](#condition) that determines the payables that users with a specific role may approve.

File: docs/w.glossary.md [hunk 6]
```diff
@@ -21,4 +21,12 @@ Learn the significance of terminology commonly used in the documentation.
...
+  - The following terms might be used as synonyms and help your understanding: Workflow / Approval flow / Spending policy
+#### Condition
+  - A setting that determines the situation for which an [Approval policy](@approval policy) should be applied.
```
Comment (billy-the-fish) [inline docs/w.glossary.md:28]: when an [Approval policy](#approval-policy) is applied.

File: docs/w.glossary.md [hunk 7]
```diff
@@ -21,4 +21,12 @@ Learn the significance of terminology commonly used in the documentation.
...
+#### Rule
+  - A setting that defines the users who have the rights to approve a submitted payable that contains figures in a specific range.
+#### Approval permissions
+  - Rights of the user to validate [payables](@payable) entered into the system.
```
Comment (billy-the-fish) [inline docs/w.glossary.md:32]: [payables](#payable)

**Review by billy-the-fish — CHANGES_REQUESTED** [review]

**Comment by Babazyga** [comment]
@billy-the-fish 

Thanks, Iain, please check the new set of fixes.

P.S. Maybe you know, why the link didn't apply to the "Approval policies" text?

**Pull request merged**